src/mmw/apps/modeling/models.py

```diff
@@ -43,6 +43,14 @@ class Project(models.Model):
         null=True,
         help_text='Serialized JSON representation of additional'
                   ' data gathering steps, such as MapShed.')
+    mapshed_job_uuid = models.UUIDField(
```

Comment:
Since this really a `ForeignKey` field, consider replacing with:

```python
mapshed_job = models.ForeignKey(
  Job,
  to_field='uuid',
  null=True,
  help_text='The job used to calculate the MapShed results. '
            'Used for getting the results of that job.'
),
```

Reply:
Same for `subbasin_job` below.